java-api-core/src/com/kinvey/java/store/requests/data/save/SaveListBatchRequest.kt

```diff
@@ -80,17 +81,18 @@ class SaveListBatchRequest<T : GenericJson>(
     }
 
     @Throws(IOException::class)
-    private fun runSaveItemsRequest(objects: Iterable<T>): List<T> {
+    private fun runSaveItemsRequest(objects: List<T>, useCache: Boolean = true): List<T> {
```

Comment:
would be better to keep `Iterable<T>` since `List<T>` is also an `Iterable<T>`

Reply:
Fixed.